install-dev/data/xml/tab.xml

```diff
@@ -626,8 +621,8 @@
                 </tab>
                 <tab id="FeatureFlag" id_parent="Advanced_Parameters" active="1" enabled="1" icon="">
                   <class_name>AdminFeatureFlag</class_name>
-                  <wording>Experimental Features</wording>
-                  <wording_domain>Admin.Navigation.Menu</wording_domain>
+                  <wording>New &amp; Experimental Features</wording>
+                  <wording_domain>Admin.Advparameters.Feature</wording_domain>
```

Comment:
```suggestion
                  <wording_domain>Admin.Navigation.Menu</wording_domain>
```

Reply:
I don't think we're supposed to change this one, even if it is indeed duplicated in the `Admin.Advparameters.Feature` it seems like all the wording for the tab have their dedicated namespace `Admin.Navigation.Menu`

This makes sense if you want to have different translated content in the menu and in the page title.

Reply:
I guess @Julievrz can confirm (or not) this namespace.